Calibration/HcalAlCaRecoProducers/src/AlCaHOCalibProducer.cc

```diff
@@ -1,5 +1,18 @@
 // -*- C++ -*-
+//integrate the code with some 8_0_X or 7_6_X recent IB and run the 
```

Comment:
@bsunanda this should be moved to plugins: it is an EDProducer used by official workflows...